tests/model/test_model.py

```diff
@@ -17,6 +17,7 @@
 
 
 CONTROLLER = 'zulipterminal.core.Controller'
+MODEL = 'zulipterminal.model.Model'
```

Comment:
I just wanted to point that I had a PR #959 that does a general simplification of patches in all tests. Let me know if I should make it review-ready if we wish to merge it first.

Reply:
@Ezio-Sarthak Thanks for reminding me of your PR :+1:

I welcome your intent to add more constants to improve readability; the main challenge with using these is that we don't currently have much of a "standard" way of applying these constants - or it seems that way, but isn't quite. In some pre-existing cases we have files referred to, others to classes, and I think this is where we would benefit from standardizing and clarying. You'll note that in my commit I only extracted the classes, since they were both the longest - so more likely to benefit from compacting the long lines - but also that it made clear that others were module-specific, either imports or not in the main class.